IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormPullTests.cs

```diff
@@ -136,9 +136,9 @@ public void Should_correctly_setup_form_title_for_fetch_action()
                 null, null, pullAction);
         }
 
-        [TestCase(AppSettings.PullAction.None, true, false, false, false, false, false, false, false)]
-        [TestCase(AppSettings.PullAction.Merge, true, false, false, false, false, false, false, false)]
-        [TestCase(AppSettings.PullAction.Rebase, false, false, false, true, false, false, false, false)]
+        [TestCase(AppSettings.PullAction.None, false, false, false, false, true, false, true, true)]
+        [TestCase(AppSettings.PullAction.Merge, false, false, false, false, true, false, true, true)]
+        [TestCase(AppSettings.PullAction.Rebase, false, false, false, false, true, false, true, true)]
```

Comment:
See above comment regarding `AppSettings.DefaultPullAction`.
I think you must extend the setup of this testcase to have a remote branch.